docs/cdp/cdp_activation.md

```diff
@@ -63,6 +66,8 @@ It will take you to a Dataflow Creator, where in five steps you will set up a da
 In the **General Information** section, specify dataflow name,
 choose **Stream File** as a source of user data and **CDP** as a destination,
 where they will be sent for processing.
+Currently, only Stream File transport is supported and can be initialized from the configuration. 
+But you can implement other methods by 3rd parties using extension point.
```

Comment:
I'd skip second sentence and add it later with the description of provided extension points. It'll probably take us some time to revisit it because there are not plan to support more transports ourselves. Also, in order to make it easier to use other transports it'd require from us to provide a few more things than just a generic configuration. In short, for now, we should just not mention it :)